src/cow-react/utils/fractionToString.ts

```diff
@@ -0,0 +1,18 @@
+import { Fraction } from '@uniswap/sdk-core'
+
+export function fractionToString(f: Fraction | null | undefined): string {
```

Comment:
we don't save that much calling the fraction `f`. Can we call it `fraction` ?

Reply:
Thank you! Fixed